script/ProtocolRewards.s.sol

```diff
@@ -0,0 +1,48 @@
+// SPDX-License-Identifier: UNLICENSED
+pragma solidity 0.8.19;
+
+import {Script} from "forge-std/Script.sol";
+import {ProtocolRewards} from "../contracts/ProtocolRewards.sol";
+import {ERC1967FactoryConstants} from "solady/src/utils/ERC1967FactoryConstants.sol";
+import {ERC1967Factory} from "solady/src/utils/ERC1967Factory.sol";
+
+// # To deploy and verify ProtocolRewards.sol run this command below
+// forge script script/ProtocolRewards.s.sol:ProtocolRewardsDeploy --rpc-url sepolia --broadcast --verify -vvvv
+contract ProtocolRewardsDeploy is Script {
```

Comment:
Is it worth writing forge tests for the scripts? If you think it is let's capture it separate and just get this in.